modules/ROOT/pages/anypoint-mq/anypoint-mq-connector-reference.adoc

```diff
@@ -47,7 +47,7 @@ November 2, 2018
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| URL a| String |  The region URL where the queue resides. This URL can be obtained and configured from the Anypoint Platform > MQ console. Copy and paste the region URL into this field. |  https://mq-us-east-1.anypoint.mulesoft.com/api/v1 |
+| URL a| String |  The region URL where the queue resides. This URL can be obtained and configured from the Anypoint Platform > MQ console. Copy and paste the region URL into this field." |  https://mq-us-east-1.anypoint.mulesoft.com/api/v1 |
```

Comment:
Copy and paste the region URL into this field."

I think the quote marks snuck in